controllers/baker.js

```diff
@@ -66,8 +66,8 @@ exports.baker = function (req, res) {
       const shouldAward = query.award && query.award !== 'false';
       awardOptions.recipient = query.award;
       awardOptions.imagedata = imageData;
-      if (shouldAward)
-        return awardBadge(awardOptions, callback)
+      // if (shouldAward)
```

Comment:
@auralon could you please add a code comment here explaining why we are switching this off? This will help us remember the reason in the future.